mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/config/WorkerConfiguration.java

```diff
@@ -170,6 +183,11 @@ default Map<String, String> getTaskExecutorAttributes() {
             return ImmutableMap.of();
         }
 
-        return Splitter.on(",").withKeyValueSeparator(':').split(input);
+        Map<String, String> attributes = Splitter.on(",").withKeyValueSeparator(':').split(input);
+
+        // filter out entries where the value matches the pattern "${.*}"
```

Comment:
can this be fixed on the input side?

Reply:
I looked into this but it's a bit tricky and worried to introduce some kinda of regression..